docs/sources/upgrade-guide/when-to-upgrade/index.md

```diff
@@ -138,12 +142,45 @@ A critical feature degradation usually meets one of the following criteria:
+2. We perform manual acceptance and smoke testing internally for new features by deploying to our internal observability stack. After that, we progressively roll out in Grafana Cloud, and then cut an OSS and Enterprise release. Each stage catches bugs.
+3. We ship new features in Experimental or Private Preview [release stages](https://grafana.com/docs/release-life-cycle/), behind feature toggles. This helps us improve the feature during development. If you are interested in gaining early access to features (including in your development or test environments), please let us know.
+4. We scan Grafana, all plugins and their dependencies continuously for security vulnerabilities.
+
+## Minimize the likelihood of bugs and problems during upgrade
+
+Despite thorough testing, you can experience problems when upgrading:
+
+### Bugs
+
+Bugs are unexpected side effects of code changes in the release, which cause problems. Some bugs occur for all users, and we usually catch these in the early stages of testing. Others occur in a small number of Grafana instances with specific configuration or unusual use cases; for example a specific authentication setup or a combination of feature toggles. Grafana plugins also interact with external services via API to query data, and sometimes these APIs change without notice, causing issues for your dashboards that depend on these datasources. Grafana Labs has monitoring in place to regularly test these APIs, but at times they break in unexpected ways.
+
+Reduce the risk of bugs by staying current and rolling out upgrades across dev or test environments before production. A Grafana Enterprise license entitles you to an additional dev and test instance for this purpose, available through your account team, and in Grafana Cloud you can create dev and test stacks that upgrade before production by using [rolling release channels](https://grafana.com/docs/rolling-release/).
+
+- Roll back and report issues if you experience problems in dev or test.
+- In Cloud, run a dev or test stack on the `fast` channel, and run your production stack on `steady` or `slow`.
+- To change your release channel, open a support ticket.
+
+### Known breaking changes
+
+As a rule we always seek backward compatibility and migration, and reserve breaking changes for Grafana’s once-yearly major release. However occasionally small breaking changes (like updates to API payloads) will ship in minor releases. These are announced in [upgrade guides](https://grafana.com/docs/grafana/latest/upgrade-guide/), [What’s New](https://grafana.com/docs/grafana-cloud/whats-new/), and our [changelog](https://github.com/grafana/grafana/blob/main/CHANGELOG.md).
+
+Always read the [upgrade guide](https://grafana.com/docs/grafana/latest/upgrade-guide/) and [changelog](https://github.com/grafana/grafana/blob/main/CHANGELOG.md) prior to upgrading to learn about and account for breaking changes.
```

Comment:
```suggestion
Always read the [upgrade guide](https://grafana.com/docs/grafana/latest/upgrade-guide/) and [changelog](https://github.com/grafana/grafana/blob/main/CHANGELOG.md) prior to upgrading to learn about and account for breaking changes.

For Grafana Cloud, we will take care of any known breaking changes for you, we will have a migration strategy and/or inform you if we are removing features. To learn more about this please read our deprecation policy.
```

Reply:
@timlevett is there a link to our deprecation policy?

Reply:
Best we have publicly is https://github.com/grafana/grafana/blob/main/contribute/breaking-changes-guide/breaking-changes-guide.md#i-know-its-a-breaking-change-whats-next i believe